README.md

```diff
@@ -16,6 +16,7 @@ The following extensions allows you to interact with some of the Azure Services:
 
 - [Quarkus Azure App Configuration Extension](https://docs.quarkiverse.io/quarkus-azure-services/dev/quarkus-azure-app-configuration.html): [Azure App Configuration](https://azure.microsoft.com/products/app-configuration)
   is a fast, scalable parameter storage for app configuration.
+- [Quarkus Azure Key Vault Extension](https://docs.quarkiverse.io/quarkus-azure-services/dev/quarkus-azure-keyvault.html): [Azure Key Vault](https://azure.microsoft.com/en-us/products/key-vault) is a cloud service for securely storing and accessing secrets.
```

Comment:
remove `en-us` in the url

Reply:
For https://github.com/quarkiverse/quarkus-azure-services/pull/218#discussion_r1582511548

With the Key Vault extension, the native build takes more time and causes authentication expired. We have to login again to delete resources.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/5b3010c3817077fc4e8e6da0ef8938fe62cd7818.